api/management/commands/loaddata.py

```diff
@@ -0,0 +1,47 @@
+import os
+import csv
+
+from django.core.management.base import BaseCommand
+
+from api.models import Disease, Sample
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
```

Comment:
Was following the [Django docs for management commands](https://docs.djangoproject.com/en/1.10/howto/custom-management-commands/#accepting-optional-arguments). The code does work, for instance using `python manage.py acquiredata --path otherdatadir`. Though perhaps I should've included some documentation and/or tests.